Dockerfile

```diff
@@ -14,10 +14,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.17.6-buster
-RUN GO111MODULE=off go get -u github.com/rexray/gocsi/csc
-
-FROM centos:7.9.2009
+FROM golang:1.17.7-buster AS go1
+RUN go install github.com/rexray/gocsi/csc@latest
+
+FROM golang:1.17.7-buster AS go2
+# Compile latest goofys for arm64 if necessary, which doesn't have a released binary
+RUN set -eux ; \
+    ARCH="$(arch)"; \
+    if [ ${ARCH} = "aarch64" ]; then \
+        git clone https://github.com/kahing/goofys.git ; \
+        cd goofys ; \
```

Comment:
Should we checkout the "v0.24.0" tag before building? Otherwise we may not get the same build as the 0.24.0 release we are downloading for x86.

Reply:
Hey @sodonnel , thanks for the comment.

Actually I tried earlier, `v0.24.0` tag wouldn't build on arm64 (in Docker on my M1 Mac). So [some master branch commits](https://github.com/kahing/goofys/compare/v0.24.0...master) probably fixed it.

Reply:
Done